### ISSUE TITLE
Dragging to darker colors resets to black

### ISSUE THREAD

**Issue by drdreo** [scttcper/ngx-color#336]
Try any color picker that allows for dragging https://ngx-color.vercel.app

If you go to the bottom right corner, it stops being accurate and will finally reset to black in the left bottom corner.

I would expect it to stay at the point where the user has dragged the cursor to. Comparing behavior to: https://zefoy.github.io/ngx-color-picker/

**Comment by scttcper** [comment]
yeah, react-color has the same issue. Can't say i'm going to fix this. Has to do with how the colors are stored
